src/hdtSkinnedMesh/hdtSkinnedMeshWorld.cpp

```
@@ -229,58 +229,51 @@ namespace hdt
-				btTypedConstraint* constraint = m_constraints[i];
-				if (constraint->isEnabled()) {
-					const btRigidBody* colObj0 = &constraint->getRigidBodyA();
-					const btRigidBody* colObj1 = &constraint->getRigidBodyB();
-
-					if (((colObj0) && (!(colObj0)->isStaticOrKinematicObject())) &&
-						((colObj1) && (!(colObj1)->isStaticOrKinematicObject()))) {
-						getSimulationIslandManager()->getUnionFind().unite((colObj0)->getIslandTag(),
-							(colObj1)->getIslandTag());
-					}
+
+		int numConstraints = int(m_constraints.size());
```

Clang (14.0.6)

[warning] variable 'numConstraints' is not initialized

(cppcoreguidelines-init-variables)